### PR TITLE
Converts README to ember-cli-addon-docs site.

### PR DESCRIPTION
**Pull request by scalvert**
As the title suggests, this PR converts the README to an `ember-cli-addon-docs` site.

Ember Exam Docs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+- `yarn lint:hbs`
+- `yarn lint:js`
+- `yarn lint:js --fix`
+
+## Running tests
+
+- `ember test` – Runs the test suite on the current Ember version
+- `ember test --server` – Runs the test suite in "watch mode"
+- `ember try:each` – Runs the test suite against multiple Ember versions
```
Comment (step2yeung) [inline CONTRIBUTING.md]: we should add `yarn test:node` to run tests from `/node-test`
and replace `ember try:each` with `yarn test:all`

File: package.json [hunk 2]
```diff
@@ -80,5 +86,6 @@
       "node-tests",
       "tests"
     ]
-  }
+  },
+  "homepage": "https://trentmwillis.github.io/ember-exam"
```
Comment (step2yeung) [inline package.json]: should this be `https://ember-cli.github.io/ember-exam`

**Comment by stefanpenner** [comment]
Very nice! Let me know when its ready to go.

File: package.json (hunk 2 above)
Comment (scalvert) [reply package.json]: Yes. Of course unrelated, but I can update.

File: CONTRIBUTING.md (hunk 1 above)
Comment (scalvert) [reply CONTRIBUTING.md]: Done!

**Comment by scalvert** [comment]
@stefanpenner I believe this is ready to do. Just follow the directions in the `ember-cli-addon-docs` documentation for deploying this, as outlined in the PR description.

TL;DR
1. Merge this PR to master
2. Tag and release
3. Run `ember deploy production`, and follow the directions (it'll ask you if you want to create a `gh-pages` branch, choose `yes`.
4. Ensure that the docs work by clicking the link in the newly updated README
5. Copy the URL for the docs site, and update the repository's website link by editing the repository name (it's a separate field beside the name)

Let me know if there's anything I've forgotten.

**Comment by scalvert** [comment]
Will rebase this with master.

**Comment by scalvert** [comment]
Seems to be an odd, perhaps unrelated, test failure in CI. Can you trigger another build?

**Comment by stefanpenner** [comment]
restarted build

**Comment by scalvert** [comment]
@stefanpenner build is green again. This thing is good to go!

File: yarn.lock [hunk 3]
```diff
@@ -3166,6 +4733,28 @@ ember-cli-broccoli-sane-watcher@^3.0.0:
...
+  resolved "https://registry.yarnpkg.com/ember-cli-clipboard/-/ember-cli-clipboard-0.11.1.tgz#caa6aaae498f12922102555d6825ad81ad843d2a"
+  integrity sha512-LAsrFpaOV8mgyI4MLR6R2BJFbzW8ac3GZkTKmAH+V5LeyidK/Inr4yZpY5nff/jKlQFT6E4991Z9mzldfKJZcg==
+  dependencies:
+    broccoli-funnel "^1.1.0"
+    clipboard "^2.0.0"
+    ember-cli-babel "^7.1.2"
+    ember-cli-htmlbars "^2.0.2"
+    fastboot-transform "0.1.1"
+
+ember-cli-clipboard@^0.8.1:
```
Comment (Turbo87) [inline yarn.lock]: duplicate `ember-cli-clipboard` dependency 🤔

Comment (scalvert) [reply yarn.lock]: Odd. Just ran `yarn-deduplicate`. Pushing that in now.

**Comment by stefanpenner** [comment]
@scalvert just rebased, will try to land once green.
